LABS-EU3/quality_hub_frontend
# Landing page Two (Only saved in repo, don't need to use right now)

##  What does this PR do
Add second landing page 
Add Modal to Feedback

##  Description of task to be completed
Build fully responsive landing page
Add modal to feedback, basic styling

##  How should this manually be tested
In App.js on line 61 change component 'Landing' to 'LandingTwo' check browser on localhost

##  What are the relevant Trello board stories
As a user i want to see a Landing page

##  Screenshots (If appropriate)


##  Questions (If any):
